.github/workflows/dotnet_pull_request.yml

```diff
@@ -15,10 +15,12 @@ jobs:
     - name: Check code formatting
       run: |
         dotnet tool restore
-        dotnet fantomas --check **/*.fs
+        dotnet fantomas --check -r src
```

Comment:
You can actually pass multiple folder `--check -r src samples templates`.
That should work as well and will be slightly faster.
See https://github.com/fsprojects/fantomas/blob/master/docs/Documentation.md#multiple-paths